roles/cs.mysql-configure/tasks/create-db.yml

```diff
@@ -1,10 +1,17 @@
+- name: Download CA RDS
```

Comment:
This is downloaded so many times. Why? Cant we just download it once ? Or check if it does not exists?